ontotext-yasgui-web-component/src/pages/view-configurations/main.js

```diff
@@ -0,0 +1,62 @@
+const ontoElement = document.querySelector("ontotext-yasgui");
+ontoElement.config = {
+  query: '',
+  yasguiConfig: {
+    requestConfig: {
+      endpoint: "/repositories/test-repo"
+    }
+  }
+};
+
+ontoElement.addEventListener("queryExecuted", () => {
+  const div = document.createElement('div');
+  div.innerHTML = '<div id="queryRan">Query was Executed</div>';
+  document.body.appendChild(div);
+});
+
+function setShowEditorTabs(hideEditorTab) {
+  ontoElement.config = {
```

Comment:
In order to prevent duplicating this again and again, you can just spread the original config and add the rest, e.g.
```
ontoElement.config = {...ontoElement.config, showEditorTabs: hideEditorTab}
```